postgres/persistent.yaml

```diff
@@ -0,0 +1,69 @@
+version: "2.0"
+services:
+  postgres:
+    image: postgres
+    params:
+      storage:
+        data:
+          mount: /var/lib/postgres
+  grafana:
+    image: grafana/grafana
+    expose:
+      - port: 3000
+        as: 80
+        to:
+          - global: true
+        accept:
+          - webperstest.localhost
+    params:
+      storage:
+        data:
+          mount: /var/lib/grafana
+profiles:
+  compute:
+    grafana-profile:
+      resources:
+        cpu:
+          units: 1
+        memory:
+          size: 1Gi
+        storage:
+          - size: 512Mi
+          - name: data
+            size: 1Gi
+            attributes:
+              persistent: true
+              class: beta2
+    postgres-profile:
+      resources:
+        cpu:
+          units: 1
+        memory:
+          size: 1Gi
+        storage:
+          - size: 512Mi
+          - name: data
+            size: 10Gi
+            attributes:
+              persistent: true
+              class: beta2
+  placement:
+    westcoast:
```

Comment:
We'd want the same `signedBy` entry as [here](https://github.com/ovrclk/awesome-akash/pull/259/files#diff-bebeb706241446634ed82840f65abf90faed5a7de543ece8887d6c8d0d890e65R24-R26)

Reply:
of course, yeah

Reply:
of course, yeah